Dc text attributes changer

vgromov opened the pull request:
This is a re-creation of PR wxDCTextBgColourChanger and wxDCTextBgModeChanger helper classes #1296
I messed up my initial fork due to lack of git workflow experience, and finally got to re-create it from scratch.
All pending proposed review changes, mentioned in #1296 discussion were implemented.

vgromov committed:
Text and Text Background mode changer helper classes added

vgromov committed:
Small documentation fixes.

PBfordev commented:
I believe that the `@since` tag should be removed from the methods' descriptions, be only in the class description.

vgromov commented:
Yep, sounds right17.04.2019, 12:33, "PB" <notifications@github.com>:I believe that the @since tag should be removed from the methods' descriptions, be only in the class description.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

-- С уважением,начальник отдела разработки ПО Эко-Е Громов Всеволод Владимирович Best regards,head of Software Development dept. Eco-EVsevolod V Gromov Skype: gromov.vsevolodMobile: +79035975260

vgromov committed:
Update dc.h

Redundant "since 3.1.3" removed from wxDCBgModeChanger and wxDCTextBgColourChanger methods' documentation

vadz commented:
Looks good, thanks! Will merge soon.
